tests/integration/components/battleship-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | battleship', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
```

Comment:
strip generic comments maybe

Reply:
Ah will probably rip out the test then. we don't need an svg test.

Reply:
a canary test that it renders at all is probably not bad. just... don't put any more logic in it than ... it renders

Reply:
cuz we want to know if it doesn't, yeah?